From 456ff65078f55746207984fe56081a63d2d3515b Mon Sep 17 00:00:00 2001
From: Sai Vemuri <svemuri@redhat.com>
Date: Thu, 11 Sep 2014 22:42:30 -0400
Subject: [ethernet] cxgb4: Remove unused registers and add missing ones

Message-id: <1410475447-94392-12-git-send-email-svemuri@redhat.com>
Patchwork-id: 93882
O-Subject: [RHEL7.1 PATCH BZ 1124947 011/108] cxgb4: Remove unused registers and add missing ones
Bugzilla: 1124947
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Remove unused registers for registers list, and add missing ones
Based on original work by Santosh Rastapur <santosh@chelsio.com>

Signed-off-by: Kumar Sanghvi <kumaras@chelsio.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
(cherry picked from commit c1f49e3e4adfba6eb953481d6731b2ee07085237)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/chelsio/cxgb4/cxgb4_main.c b/drivers/net/ethernet/chelsio/cxgb4/cxgb4_main.c
index 6321ddd..950dea1 100644
--- a/drivers/net/ethernet/chelsio/cxgb4/cxgb4_main.c
+++ b/drivers/net/ethernet/chelsio/cxgb4/cxgb4_main.c
@@ -2064,7 +2064,7 @@ static void get_regs(struct net_device *dev, struct ethtool_regs *regs,
   0x40200, 0x40298,
   0x402ac, 0x4033c,
   0x403f8, 0x403fc,
-  0x41300, 0x413c4,
+  0x41304, 0x413c4,
   0x41400, 0x4141c,
   0x41480, 0x414d0,
   0x44000, 0x44078,
@@ -2092,7 +2092,7 @@ static void get_regs(struct net_device *dev, struct ethtool_regs *regs,
   0x48200, 0x48298,
   0x482ac, 0x4833c,
   0x483f8, 0x483fc,
-  0x49300, 0x493c4,
+  0x49304, 0x493c4,
   0x49400, 0x4941c,
   0x49480, 0x494d0,
   0x4c000, 0x4c078,
-- 
1.7.1